app/src/main/java/com/mashup/network/dao/DanggnRankDao.kt

```diff
@@ -0,0 +1,31 @@
+package com.mashup.network.dao
+
+import com.mashup.data.dto.DanggnAllMemberRankResponse
+import com.mashup.data.dto.DanggnMemberRankResponse
+import com.mashup.data.dto.DanggnPlatformRankResponse
+import com.mashup.network.Response
+import retrofit2.http.GET
+import retrofit2.http.Query
+
+/**
+ * [swagger] https://api.dev-member.mash-up.kr/swagger-ui/index.html#/danggn-controller/getMemberRankUsingGET
+ */
+interface DanggnRankDao {
+    // 당근 흔들기 개인별 랭킹
+    @GET("api/v1/danggn/rank/member")
+    suspend fun getDanggnMemberRank(
+        @Query("generationNumber") generationNumber: Int, @Query("limit") limit: Int
+    ): Response<DanggnMemberRankResponse>
+
+    // 당근 흔들기 개인별 랭킹 전체
+    @GET("api/v1/danggn/rank/member/all")
+    suspend fun getDanggnAllMemberRank(
+        @Query("generationNumber") generationNumber: Int
+    ): Response<DanggnAllMemberRankResponse>
```

Comment:
```suggestion
    ): Response<List<MemeberRankResponse>>
```

여기도 이렇게 정의하면 될 것 같아요!! 나머지도 부탁드립니당

Reply:
[33c611d](https://github.com/mash-up-kr/mashup_Android/pull/272/commits/33c611d303a4fb0d81e47d1cd8bf0b8d2957a73b)